src/avk.cpp

```diff
@@ -2,6 +2,11 @@
 #include <avk/avk_log.hpp>
 #include "avk/avk.hpp"
 
+#ifdef AVK_USES_VMA
+#define VMA_IMPLEMENTATION
+#include <vma/vk_mem_alloc.h>

```

Comment:
When installing the SDK on Ubuntu, vk_mem_alloc.h is located at /usr/include/vk_mem_alloc.h.
See comment above